test-requirements.txt

```diff
@@ -1,5 +1,6 @@
 ansible-core
 black
+curses-2048
```

Comment:
Why did you add this?

Reply:
I thought it would fix one of the pytest failures, but apparently not